css/main.css

```diff
@@ -153,17 +156,220 @@ nav {
   line-height: 24px;
   color: #7f8cff;
   margin-top: 12px;
+  text-transform: uppercase;
 }
 
 .ellipse12 {
-  left: 420px;
-  top: 474px;
+  left: 317.67px;
+  top: 490px;
   z-index: 10;
 }
 
 .ellipse13 {
-  left: 410px;
-  top: 485px;
+  left: 304px;
+  top: 500px;
 }
 
 /* END OF INTRODUCTION */
+
+/* Work section */
+
+.worksection {
+  min-width: 375px;
+  margin: 100px 0;
+  display: grid;
+  grid-template-columns: 1fr;
+  place-items: center;
+  min-height: 100vh;
+  gap: 80px;
+}
+
+.card {
+  display: flex;
```

Comment:
Okay, I will work on this immediately. Thank you very much

Reply:
Thank you so much for the observation. .card is a class defining rules holding the all the cards. I defined display:flex in different rules because without it other flex properties won't work.